src/store/getters.js

```diff
@@ -159,5 +159,4 @@ export const getters = {
 	isFollowUpFeatureAvailable: (state) => state.followUpFeatureAvailable,
 	getInternalAddresses: (state) => state.internalAddress?.filter(internalAddress => internalAddress !== undefined),
 	hasCurrentUserPrincipalAndCollections: (state) => state.hasCurrentUserPrincipalAndCollections,
-	showSettingsForAccount: (state) => (accountId) => state.showAccountSettings === accountId,
```

Comment:
Please restore the line because it's still needed ;)

Reply:
thats a mistake of the rebase not on purpose of course :(